BPOR/BPOR.Registration.Stream.Handler/src/BPOR.Registration.Stream.Handler/Mappers/ParticipantMapper.cs

```diff
@@ -136,19 +154,23 @@ public async Task<Participant> Map(Dictionary<string, AttributeValue> record, Pa
             destination.SourceReferences.Add(new SourceReference { Pk = record.PK() });
         }
 
-        ParticipantAddressMapper.Map(source.Address, destination);
-        var coordinates =
-            await _locationApiClient.GetCoordinatesFromPostcodeAsync(source.Address.Postcode, cancellationToken);
+        if(source.Address != null) { 
+            ParticipantAddressMapper.Map(source.Address, destination);
 
-        if (coordinates != null)
-        {
-            destination.ParticipantLocation ??= new ParticipantLocation();
-            destination.ParticipantLocation.Location = new Point(coordinates.Longitude, coordinates.Latitude)
+            var coordinates =
+                await _locationApiClient.GetCoordinatesFromPostcodeAsync(source.Address.Postcode, cancellationToken);
```

Comment:
I haven't traced this, if this is an external api client, do we need to consider error codes/failure scenarios? or is it covered?

Reply:
the method checks `response.IsSuccessStatusCode`, this covers the vast majority of issues that would be seen with the api call.